app/views/transferAgreementAlreadyCompleted.scala.html

```diff
@@ -0,0 +1,104 @@
+                Symbol("_value") -> "true",
+                Symbol("_smallCheckbox") -> true,
+                Symbol("_checkedOption") -> "true",
+                Symbol("_requiredOption") -> true,
+                Symbol("_disabledOption") -> "true"
+            )
+            @inputSingleCheckbox(
+                transferAgreementForm("crownCopyright"),
+                Symbol("_label") -> "I confirm that the records are all Crown Copyright.",
+                Symbol("_value") -> "true",
+                Symbol("_smallCheckbox") -> true,
+                Symbol("_checkedOption") -> "true",
+                Symbol("_requiredOption") -> true,
+                Symbol("_disabledOption") -> "true"
+            )
+            @inputSingleCheckbox(
+                transferAgreementForm("english"),
+                Symbol("_label") -> "I confirm that the records are all in English.",
+                Symbol("_value") -> "true",
+                Symbol("_smallCheckbox") -> true,
+                Symbol("_checkedOption") -> "true",
+                Symbol("_requiredOption") -> true,
+                Symbol("_disabledOption") -> "true"
+            )
+            <!-- DRO confirmation -->
+            @inputSingleCheckbox(
+                transferAgreementForm("droAppraisalSelection"),
+                Symbol("_label") -> "I confirm that the Departmental Records Officer (DRO) has signed off on the appraisal and selection decision.",
+                Symbol("_value") -> "true",
+                Symbol("_smallCheckbox") -> true,
+                Symbol("_checkedOption") -> "true",
+                Symbol("_requiredOption") -> true,
+                Symbol("_disabledOption") -> "true"
+            )
+            @inputSingleCheckbox(
+                transferAgreementForm("droSensitivity"),
+                Symbol("_label") -> "I confirm that the Departmental Records Officer (DRO) has signed off on the sensitivity review.",
+                Symbol("_value") -> "true",
+                Symbol("_smallCheckbox") -> true,
+                Symbol("_checkedOption") -> "true",
+                Symbol("_requiredOption") -> true,
+                Symbol("_disabledOption") -> "true"
+            )
+            @inputSingleCheckbox(
+                transferAgreementForm("openRecords"),
+                Symbol("_label") -> "I confirm that all records are open and no Freedom of Information (FOI) exemptions apply to these records.",
+                Symbol("_value") -> "true",
+                Symbol("_smallCheckbox") -> true,
+                Symbol("_checkedOption") -> "true",
+                Symbol("_requiredOption") -> true,
+                Symbol("_disabledOption") -> "true"
+            )
+        </div>
+        <!-- Buttons -->
+        <div class="govuk-button-group">
+            <!-- Continue -->
+            <button data-prevent-double-click="true" class="govuk-button" type="submit" data-module="govuk-button" role="button">
+                Continue
```

Comment:
Should this one be a link rather than a Submit button? What happens if you click it at the moment? Does it try to re-save the the form data?

Reply:
Yes, that would make more sense, although, if I keep just the one TA page, and use logic to display the banner and greyed out buttons, I'd have to keep it this way